web/assets/css/error.css

```diff
@@ -0,0 +1,42 @@
+.content-error-page .content h1 {
+    font-family: 'Maven Pro', sans-serif;
+    font-size: 90px;
+    font-weight: 900;
+    color: #63605d;
+    margin: 106px 0 0 0;
+    text-transform: uppercase;
+}
+
+.content-error-page .content h2 {
+    font-family: 'Maven Pro', sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+    color: #63605d;
+    padding: 45px 0;
+    margin: 0;
+    text-transform: uppercase;
+}
+
+.content-error-page a.goback {
```

Comment:
This kind of outline button will be used on other pages
Please create ```.btn-outline``` on https://github.com/ezsystems/ezplatform-com/blob/master/web/assets/css/ezplatform-com.css